<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="addLengLine">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addLengButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить язык</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delLengButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить выбранный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modelCounterButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество строк в модели</string>
    </property>
   </widget>
   <widget class="QLabel" name="modelItemCounter">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="modelColumnsButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество колонок</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="viewModeBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отображать в виде иконок</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
